contracts/pre-propose/dao-pre-propose-approver/src/contract.rs

```diff
@@ -188,6 +188,9 @@ pub fn query(deps: Deps, env: Env, msg: QueryMsg) -> StdResult<Binary> {
             QueryExt::PreProposeApprovalContract {} => {
                 to_binary(&PRE_PROPOSE_APPROVAL_CONTRACT.load(deps.storage)?)
             }
+            QueryExt::PendingProposalIdForApprovalProposalId { id } => {
+                to_binary(&PROPOSAL_IDS.load(deps.storage, id)?)
```

Comment:
Nitpicking outside of the pr scope but I would rename that `PROPOSAL_IDS` map to something a little more obvious, like `PROPOSAL_ID_TO_PRE_PROPOSE_ID_MAP`

Reply:
Shall we do a `may_load` here to avoid errors in case it's not set?

Reply:
Agree with the `may_load`.

Reply:
> Shall we do a `may_load` here to avoid errors in case it's not set?

sounds good